config/schema/synonyms.yml

```diff
@@ -269,6 +269,7 @@
 - search: british citizenship => british citizenship, british citizen
 - search: health check, health checks => nhs health check
 - search: job share, job sharing => job share, job sharing, flexible working
+- search: COVID-19, covid19, covid, coronavirus => COVID-19, coronavirus
```

Comment:
Just for consistency (and because everything gets lowercased anyway), I think the first term should be `covid-19`.